src/formats/images/jpg.js

```
@@ -0,0 +1,837 @@
+      var j, k, ll = 0;
+      var lineOffset = 0;
+
+      for (let blockRow = 0; blockRow < blocksPerColumn; blockRow++) {
+        const scanLine = blockRow << 3;
+
+        for (let blockCol = 0; blockCol < blocksPerLine; blockCol++) {
+          const bufferOffset = getBlockBufferOffset(component, blockRow, blockCol);
+          let offset2 = 0, sample = blockCol << 3;
+
+          for (j = 0; j < 8; j++) {
+            var lineOffset = (scanLine + j) * samplesPerLine;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'lineOffset'. Consider to delete it or rename it.

'lineOffset' is defined here:


(lint/suspicious/noRedeclare)